src/components/NavBar.js

```diff
@@ -1,19 +1,17 @@
-/* eslint-disable */
 import React from 'react';
 import { Link } from 'react-router-dom';
 
-const Navbar = () => {
-    return (
-    <div className="navBar">
-      <div className="Logo">
-        Book Store 
-      </div>
-      <ul className="links">
-        <Link className="link" to="/">Book List</Link>
-        <Link className="link" to="/category">Category</Link>
-      </ul>
-    </div>
-    )
-  }
+const Navbar = () => (
+  <div className="navBar">
+    <span className="Bookstore-CMS Text-Style-5">
+      ICAN Bookstore CMS
+    </span>
+    <ul className="links">
+      <Link className="link" to="/">Book List</Link>
+      <Link className="link" to="/category">Category</Link>
+    </ul>
+    <div className="avatar">Avator</div>
+  </div>
```

Comment:
- [ ] Please let's style the `Navbar` as closely as possible to the given design. Please remove the shadow around the navigation.
- [ ] The border color should be `#e8e8e8`
- [ ] Use a user icon on the right of the navbar just like the design guideline
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/202619705-a402647d-d228-4f3a-b4a2-a6fae73fc9ff.png)
### Your Implementation
![image](https://user-images.githubusercontent.com/46839548/202620109-22dcdbae-a8c2-41a0-8772-270728427070.png)

Reply:
Thanks for the constructive feedback! 
Issue resolved.